src/contexts/ThemeContext.jsx

```diff
@@ -5,7 +5,13 @@ import { Theme } from '../ui/themes';
 const ThemeContext = createContext();
```

Comment:
I think I missed the class where you went over "context", can you share a good easy to understand resource about it?

Reply:
Hey, check out [Web Dev Simplified](https://www.youtube.com/watch?v=5LrDIWkK_Bc&t=698s), he has a ton of really great videos on how a context is set up. I followed along with this video as I set up the `UserContext` and `ThemeContext` in our project.

If you want to pair on it as well, just let me know!